src/lectern-client/schema-entities.ts

```diff
@@ -106,7 +106,7 @@ export interface SchemaValidationError {
   readonly errorType: SchemaValidationErrorTypes;
   readonly index: number;
   readonly fieldName: string;
-  readonly info: object;
+  readonly info: any;
```

Comment:
I guess you can define this as `Record<string, any>`, `any` is too wide for this field

Reply:
yea fair point